sympy/crypto/crypto.py

```diff
@@ -2246,3 +2246,202 @@ def decipher_gm(message, key):
         m <<= 1
         m += not b
     return m
+
+################ Blum–Goldwasser cryptosystem  #########################
+
+def _pow_mod(a, b, m):
```

Comment:
How about using Python's [builtin pow](https://docs.python.org/3.7/library/functions.html#pow)?

Reply:
Thanks for this. I am going to update.